src/main/java/io/kestra/plugin/kestra/triggers/DetectStuckOrDisabledSchedules.java

```diff
@@ -0,0 +1,172 @@
+                """
+        )
+    }
+)
+public class DetectStuckOrDisabledSchedules extends AbstractKestraTask
+    implements RunnableTask<DetectStuckOrDisabledSchedules.Output> {
+
+    @PluginProperty
+    private String namespace;
+
+    @Schema(title = "Threshold in minutes to consider a schedule overdue.")
+    @Builder.Default
+    private Property<Integer> thresholdMinutes = Property.ofValue(60);
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        KestraClient kestraClient = kestraClient(runContext);
+        TriggersApi triggersApi = kestraClient.triggers();
+
+
+        int threshold = runContext.render(this.thresholdMinutes).as(Integer.class).orElse(60);
+        String tenantId = runContext.flowInfo().tenantId();
+        String namespaceToUse = this.namespace != null ? this.namespace : runContext.flowInfo().namespace();
+
+        runContext.logger().info(
+            "🔍 Detecting stuck or disabled schedule triggers (tenant='{}', namespace='{}', threshold={} min)",
+            tenantId, namespaceToUse, threshold
+        );
+
+        List<String> issues = new ArrayList<>();
+        Instant now = Instant.now();
+
+        int page = 1;
+        int size = 100;
+        boolean more = true;
+
+
+        while (more) {
+            PagedResultsTriggerControllerTriggers triggersResponse = triggersApi.searchTriggers(
+                page,
+                size,
+                tenantId,
+                null,
+                null,
+                null,
+                namespaceToUse,
+                null,
+                null
+            );
+
+            List<TriggerControllerTriggers> results = triggersResponse.getResults();
+            int returned = results.size();
+            runContext.logger().info("📦 Trigger API returned {} triggers (page {})", returned, page);
+
+            if (results.isEmpty()) {
+                break;
+            }
+
+            for (TriggerControllerTriggers trigger : results) {
+
+                if (trigger.getAbstractTrigger() == null ||
+                    !trigger.getAbstractTrigger().getType().contains("Schedule")) {
+                    continue;
+                }
+                var context = trigger.getTriggerContext();
+                if (context == null) continue;
+
+                String flowNamespace = context.getNamespace();
+                String flowId = context.getFlowId();
+                String triggerId = context.getTriggerId();
+                boolean disabled = Boolean.TRUE.equals(context.getDisabled());
+                Instant lastExec = context.getDate() != null ? context.getDate().toInstant() : null;
+
+                String name = flowNamespace + "." + flowId + "#" + triggerId;
+
+                // 🚫 Disabled triggers
+                if (disabled) {
+                    issues.add(name + " (🚫 Disabled trigger)");
+                    continue;
+                }
+
+                if (lastExec != null) {
```

Comment:
A stuck trigger is detected not based on a thresold and the lastExecution but based on the `nextExecutionDate` of the trigger. The thresold is here to avoid warning for stuch trigger too quickly as we want to allow for ex to warn only if the schedule is missing since 5mn.